### PR TITLE
GirlsCode MK 1st Blog Post

### PR DESCRIPTION
**Pull request by edieblu**

### CODE REVIEW

File: blog/_posts/2018-07-08-4Ps-of-GirlsCode-MK.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
+---
+title: Team GirlsCode MK - The 4 Ps of GirlsCode MK
```
Comment (v-thakkar) [inline blog/_posts/2018-07-08-4Ps-of-GirlsCode-MK.md]: Hi @edieblu ! Thanks for the PR. Please check the comments below.

File: blog/_posts/2018-07-08-4Ps-of-GirlsCode-MK.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+layout: post
+created_at: Sun Jul 08 2018
+permalink: blog/_posts/2018-07-08-4ps-of-girlscode-mk
```
Comment (v-thakkar) [inline blog/_posts/2018-07-08-4Ps-of-GirlsCode-MK.md]: Here, permalink need to be same as file name. Please check the old blog posts for an example.

File: blog/_posts/2018-07-08-4Ps-of-GirlsCode-MK.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+author: Team GirlsCode MK
+twitter: GirlsCodeMK
```
Comment (v-thakkar) [inline blog/_posts/2018-07-08-4Ps-of-GirlsCode-MK.md]: The file is missing bunch of parameter declarations like 'current' and 'categories'. Please add them.

File: blog/_posts/2018-07-08-4Ps-of-GirlsCode-MK.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+
+## 4. The Plan
+Our primary task is to add a person module to Bahmni. Currently it’s only possible to register people as patients in Bahmni, but we would like to add the functionality of adding persons - for example to register people for pre-screening appointments.
+**Technologies used:** React.js, Node.js, RESTful APIs, PWA
+
+<br>
+
+## You can follow our journey here:
+
+__[@GirlsCodeMK](https://twitter.com/GirlsCodeMK)__
```
Comment (v-thakkar) [inline blog/_posts/2018-07-08-4Ps-of-GirlsCode-MK.md]: May be it would be good to write whose twitter accounts these are. You can have your names as a text [linking to twitter link] instead of twitter handles.

**Review by v-thakkar — CHANGES_REQUESTED** [review]

File: blog/_posts/2018-07-08-4Ps-of-GirlsCode-MK.md (hunk 4 above)
Comment (edieblu) [reply blog/_posts/2018-07-08-4Ps-of-GirlsCode-MK.md]: Thank you Vaishali :)

**Review by v-thakkar — APPROVED** [review]

**Comment by edieblu** [comment]
@nerdyvaishali Wohooo 👍

File: Gemfile.lock [hunk 5]
```diff
@@ -125,9 +125,12 @@ PLATFORMS
 
 DEPENDENCIES
   geocoder
-  github-pages (= 84)
```
Comment (v-thakkar) [inline Gemfile.lock]: Hi @edieblu ! Usually we keep do change per PR. So, please send Gemfile changes in a separate PR if you wish to send it.

**Review by v-thakkar — CHANGES_REQUESTED** [review]

File: Gemfile.lock (hunk 5 above)
Comment (edieblu) [reply Gemfile.lock]: Hm, not sure how that got there @nerdyvaishali. Will fix it now :)

**Review by edieblu — COMMENTED** [review]
Hopefully, it's okay now. Will try not to do the "git add ." again :)

**Review by v-thakkar — APPROVED** [review]
